ComponentTests/CompressorTest.py

```diff
@@ -1,26 +1,34 @@
-from Compressor import CompressorClass
+from ACHP.Compressor import CompressorClass
+import CoolProp as CP
 from CoolProp.CoolProp import PropsSI
 
+Ref = 'R134a'
+Backend = 'HEOS' #choose between: 'HEOS','TTSE&HEOS','BICUBIC&HEOS','REFPROP','SRK','PR'
+AS = CP.AbstractState(Backend, Ref)
+    
 kwds={
       'M':[217.3163128,5.094492028,-0.593170311,4.38E-02,-2.14E-02,1.04E-02,7.90E-05,-5.73E-05,1.79E-04,-8.08E-05],
       'P':[-561.3615705,-15.62601841,46.92506685,-0.217949552,0.435062616,-0.442400826,2.25E-04,2.37E-03,-3.32E-03,2.50E-03],
-      'Ref':'R134a',
+      'Ref':Ref,
```

Comment:
This still needed now that you use AS?

Reply:
it is used to determine the solubility of refrigerant in oil.